.travis.yml

```diff
@@ -9,4 +9,8 @@ cache:
   directories:
     - "node_modules"
 
+branches:
+  only:
+  - master
+
```

Comment:
What's the effect of this?

Reply:
i had enabled travis to run when branches are updated, this tells travis to only run those for the master branch. so travis should run on PR updates as expected, and run against master after PRs are merged in. unless i misunderstood how this config works, the documentation wasn't super clear